### PR TITLE
Fixes #69 Make Minimal UI

### LINKED ISSUE #69: Make minimal UI
The only thing we need to get started is a UI like the Android-UI. We don't need any side or anything else. Please as a first start implement a minimalistic U, e.g. similar to the Android app.

f538315a-f54a-11e6-9084-9f9d73596a0c

### PR DESCRIPTION
**Pull request by rishiraj824**
**Screenshots**
susi surge sh- iphone 6
screenshot from 2017-05-27 09 23 35

**Demo**
http://susi.surge.sh

@mariobehling @isuruAb @uday96 Please review.

### CODE REVIEW

**Comment by madhavrathi** [comment]
@rishiraj824 
screenshot from 2017-05-27 10-11-20
The scrollbar seems to go below the limit of chat screen please have a look at it.
Otherwise great UI

**Comment by uday96** [comment]
screenshot from 2017-05-27 10 20 48

@rishiraj824 The marker doesnt seem to render in the map tile and the table is overflowing a bit. We could create seperate issues for that maybe, and the scrollbar goes below limit as @madhavrathi mentioned.

Apart from that the UI looks cool especially in the mobile.  :+1:

**Comment by AnshulMalik** [comment]
Looks like a step closer to PWA. Great ! @rishiraj824

**Comment by rishiraj824** [comment]
localhost-3000- iphone 6 1
localhost-3000- iphone 6
@madhavrathi @uday96 Please check once again :)
@isuruAb @AnshulMalik thanks :+1:

File: src/components/ChatApp.css [hunk 1]
```diff
@@ -6,26 +6,73 @@
...
-.message-list, .thread-list {
-  border: 1px solid #38c3ea;
-  font-size: 16px;
-  height: 400px;
-  margin: 0;
+.message-list {
+  background: #eceff1;
+  position: fixed;
+  top: 56px;
+  left: 0;
```
Comment (AnshulMalik) [inline src/components/ChatApp.css]: We have `margin: 0 auto` to place it in middle, still using `left:0` and `right:0` ?

**Pull request merged**